template/README.md

```diff
@@ -22,7 +22,10 @@ You may edit the `99-override.hcl` or add your own.
 Any valid configuration added to these directories will be added to their respective services' configuration, in lexical order.
 
 #### Nomad ACLs
-Refer to [config variations](#config-variations) on how to toggle ACLs in nomad.
+To toggle ACLs in nomad you need to supply a `env.yml` file in the same directory as your Vagrantfile containing the line:
+```yaml
+nomad_acl: true
+```
```

Comment:
Not loving this TBH. We should have _one_ file to change. Something from here maybe? https://stackoverflow.com/questions/60209185/ansible-environment-variables-from-env-file

Reply:
Not sure I understand. The way I'm making it now we will have a `env_defaults.yml` and a `env_override.yml`, where you can add any variables you want to change/override in `env_override.yml`.

Reply:
After deliberation offline: We will use .env files and source in shell.